cynthia_websites_mini_client/src/cynthia_websites_mini_client/pottery/molds/ownit_ffi.ts

```
@@ -0,0 +1,60 @@
+// Ownit layout FFI module
+// Ownit is the only layout that has it's own FFI implementations, since Gleam doesn't have any direct bindings to Handlebars.js
+// And well, I'd like to support full Handlebars :shrug:
+
+import Handlebars from "handlebars";
+import { Ok, Error } from "../../../../prelude";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)